src/commands/a32nx/CPDLC.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+export const CPDLC: CommandDefinition = {
+    name: ['cpdlc', 'pdc', 'hoppie', 'acars'],
+    description: 'Provide information about the fix info feature.',
```

Comment:
This seems to be an incorrect description.

Reply:
whoops! fixed